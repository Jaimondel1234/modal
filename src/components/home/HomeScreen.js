import React from "react";
import { PanelStyle, TextStyle, ContainerStyle } from "../../styles/core";

/*
  Component that displays the home screen
*/

export const HomeScreen = () => {
  return (
    <ContainerStyle m='5'>
      <PanelStyle bg='#1E90FF' color='#ffff' borderRadius={10} p={5}>
        <TextStyle
          fontSize='60px'
          fontWeight='bold'
          textShadow='2px 2px 4px #000000'
          textAlign='center'
        >
          Welcome to the Notes App
        </TextStyle>
      </PanelStyle>
    </ContainerStyle>
  );
};
