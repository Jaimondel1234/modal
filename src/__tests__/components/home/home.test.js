import React from 'react';
import { shallow } from 'enzyme';
import { HomeScreen } from '../../../components/home/HomeScreen';

describe('Test <HomeScreen/>', () => {
  const wrapper = shallow(<HomeScreen />);
  test('it must be displayed correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('the welcome text must appear', () => {
    expect(wrapper.find('core__TextStyle').text()).toBe(
      'Welcome to the Notes App',
    );
  });
});
