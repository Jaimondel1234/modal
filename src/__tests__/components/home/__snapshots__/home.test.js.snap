// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HomeScreen/> it must be displayed correctly 1`] = `
<core__ContainerStyle
  m="5"
>
  <core__PanelStyle
    bg="#1E90FF"
    borderRadius={10}
    color="#ffff"
    p={5}
  >
    <core__TextStyle
      fontSize="60px"
      fontWeight="bold"
      textAlign="center"
      textShadow="2px 2px 4px #000000"
    >
      Welcome to the Notes App
    </core__TextStyle>
  </core__PanelStyle>
</core__ContainerStyle>
`;
